src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php

```diff
@@ -24,6 +26,24 @@ public function getEventTransferIds(array $eventTransfers)
         return array_unique($ids);
     }
 
+    /**
+     * @param array<\Generated\Shared\Transfer\EventEntityTransfer> $eventTransfers
+     *
+     * @return array<int, int>
+     */
+    public function getEventTransferIdsWithTimestamps(array $eventTransfers): array
```

Comment:
Lots of duplicated code, which can also be eliminated by using one method and the Request/Response transfer pattern for the Facade method.